node-ui/src/api/dataSource/NodeDataSource.ts

```diff
@@ -17,6 +17,7 @@ export enum Network {
   ARB = 'ARB',
   ZK = 'ZK',
   STARKNET = 'STARKNET',
+  INTERNETCOMPUTER = 'INTERNETCOMPUTER',
```

Comment:
INTERNET_COMPUTER

Reply:
ICP maybe? 😂

Reply:
We mentioned in previous implementations that we will use full names of protocols, that is why I went with full name.